### PR TITLE
Update entity_attribute.tsv

### PR DESCRIPTION
**Pull request by rays22**
This commit is related to #52.

### CODE REVIEW

File: src/patterns/dosdp-patterns/bone_length_from_to.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+pattern_name: bone_length_from_to
```
Comment (matentzn) [inline src/patterns/dosdp-patterns/bone_length_from_to.yaml:2]: does this belong in the PR?

File: src/patterns/definitions.owl [hunk 2]
```diff
@@ -47657,6 +47674,83 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
...
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute"^^xsd:string) <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/OBA_2040142> "head of width"^^xsd:string)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/OBA_2040142> "head width"^^xsd:string)
+EquivalentClasses(<http://purl.obolibrary.org/obo/OBA_2040142> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/UBERON_0000033> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> <http://purl.obolibrary.org/obo/PATO_0000921>)))
+
+# Class: <http://purl.obolibrary.org/obo/OBA_2040143> (metafemur length)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/OBA_2040143> "The http://purl.obolibrary.org/obo/HAO_0001140 of a length."^^xsd:string)
```
Comment (matentzn) [inline src/patterns/definitions.owl:47686]: You need to run the make pipeline again after you created added the import.

File: src/patterns/data/default/entity_attribute.tsv [hunk 3]
```diff
@@ -7241,3 +7241,15 @@ OBA:1001089	multicellular organism sex	UBERON:0000468	multicellular organism	PAT
 OBA:1001090	feather color	UBERON:0000022	feather	PATO:0000014	color
 OBA:1001091	egg shell trait	UBERON:0005079	eggshell	PATO:0000001	quality
 OBA:1001092	egg shell color	UBERON:0005079	eggshell	PATO:0000014	color
+OBA:2040142	head width	PATO:0000921	width	UBERON:0000033	head
+OBA:2040143	metafemur length	PATO:0000122	length	HAO:0001140	metafemur
```
Comment (matentzn) [inline src/patterns/data/default/entity_attribute.tsv:7245]: We should probably add Uberon IDs for all OBA terms as @cmungall suggests on slack (uberon channel)

File: src/patterns/dosdp-patterns/bone_length_from_to.yaml (hunk 1 above)
Comment (rays22) [reply src/patterns/dosdp-patterns/bone_length_from_to.yaml:2]: No. It should be in another github branch. I am not sure why it shows up here.

File: src/patterns/data/default/entity_attribute.tsv (hunk 3 above)
Comment (rays22) [reply src/patterns/data/default/entity_attribute.tsv:7245]: The HAO classes are not vertebrate anatomy terms though.

File: src/patterns/dosdp-patterns/bone_length_from_to.yaml (hunk 1 above)
Comment (rays22) [reply src/patterns/dosdp-patterns/bone_length_from_to.yaml]: I have deleted the file from this github branch/

**Comment by matentzn** [comment]
What is the plan regarding this PR?

**Comment by rays22** [comment]
@matentzn : I have made a new pull request here <https://github.com/obophenotype/bio-attribute-ontology/pull/95>
, because there are too many conflicts in this one.

**Pull request closed without merging**